language.cto

```diff
@@ -0,0 +1,483 @@
+namespace org.accordproject.language
+
+/**
+ * Model to define the governing language
+ */
+ concept GoverningLanguage
```

Comment:
This contract specific concept doesn't belong in a general namespace like `org.accordproject.language`. Please remove from this file and introduce into a new namespace which imports the language type.